amd_openvx_extensions/amd_rpp/include/vx_ext_rpp.h

```diff
@@ -1053,6 +1053,19 @@ extern "C"
 	 */
 	SHARED_PUBLIC vx_node VX_API_CALL vxRppResize(vx_graph graph, vx_tensor pSrc, vx_tensor pSrcRoi, vx_tensor pDst, vx_array pDstWidth, vx_array pDstHeight, vx_scalar interpolationType, vx_scalar inputLayout, vx_scalar outputLayout, vx_scalar roiType);
 
+	/*!
+	 * \brief [Graph] Creates a Tensor SequenceRearrange function node.
```

Comment:
@fiona-gladwin we should probably not say "tensor" if we plan to get rid of all image later. It's confusing

Reply:
In RPP vx extensions both the image and tensor API would co exist